types/chrome-apps/index.d.ts

```diff
@@ -9352,6 +9352,11 @@ declare namespace chrome {
             height: integer;
         }
 
+        enum ActiveState {
+            ACTIVE = "active",
```

Comment:
Enum style in this file used ' instead of "

Reply:
Done